CHANGELOG.md

```diff
@@ -25,6 +29,12 @@ Branch: `nilrt/master/next`
 - [Added](https://github.com/ni/meta-nilrt/pull/717) `libpwquality` to the core packagefeed.
 - [Added](https://github.com/ni/meta-nilrt/pull/726) `nftables` to runmode.
 - [Added](https://github.com/ni/meta-nilrt/pull/729) `nilrt-snac` recipe to put system into SNAC configuration.
+- [Added](https://github.com/ni/meta-nilrt/pull/732) onboard virtual keyboard for use in touch-screen devices.
```

Comment:
From customer's perspective, onboard was always present in our released BSIs. The fact that it was removed in scarthgap and later add is irrelevant. So no need to mention this.

Reply:
Fixed.